components/brave_blank_page/resources/brave_blank_page.html

```diff
@@ -0,0 +1,10 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <title>Blank page</title>
+</head>
+
+<body style="background-color:#FFFFFF;">
```

Comment:
We can put a `<style>` element here which varies the background-color based on dark / light theme. Similar to on the dashboard NTP: https://github.com/brave/brave-core/blob/master/components/brave_new_tab_ui/brave_new_tab.html

```
@media (prefers-color-scheme: dark) {
    body { background-color: #333639; }
  }
```

Then we don't need _both_ `blank_page.html` and `dark_blank_page.html`.

Reply:
Fixed :+1: